components/org.wso2.carbon.business.rules.core/src/gen/java/org/wso2/carbon/business/rules/api/BusinessRuleApi.java

```diff
@@ -0,0 +1,90 @@
+package org.wso2.carbon.business.rules.api;
+
+import org.wso2.carbon.business.rules.api.factories.BusinessRuleApiServiceFactory;
+import org.wso2.carbon.business.rules.model.BusinessRule;
+import org.wso2.carbon.business.rules.model.BusinessRuleDeploymentStatus;
+
+import io.swagger.annotations.ApiParam;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.*;
```

Comment:
here too

Reply:
These are auto generated classes by swagger. 
@ramindu90 : Should we change the scope?  ( how it happened in simulator)